src/cq_cam/routers.py

```diff
@@ -211,23 +213,24 @@ def route_wires(job: "Job", wires: List[Union[cq.Wire, cq.Edge]], stepover=None)
         else:
             distance, target, param = None, None, None
 
-        if (
-            ep
-            and isclose(ep.x, start.x, abs_tol=0.001)
-            and isclose(ep.y, start.y, abs_tol=0.001)
-        ):
-            commands.append(Plunge.abs(start.z, arrow=True, feed=job.feed))
-        elif stepover and distance and distance <= stepover:
+        # This check is only relevant for the pocket operation as it is the only one that sets the stepover.
+        # FIXME I think this check here is included to solve the upstream bug with Offset2D. Remove if/else when upstream PR is accepted and we update to that version of cadquery
+        # TODO Create PR in upstream cadquery with the fix
+        if stepover and distance and distance <= stepover:
             # Determine the index of the edge, shift edges and use param?
             edges = shift_edges(edges, target)
             start = edge_start_point(edges[0])
             if param:
                 start = edges[0].positionAt(param, "parameter")
             commands.append(Cut(AbsoluteCV.from_vector(start), feed=job.feed))
         else:
```

Comment:
This is the fallback dumb routing method, which will do a full retract, reposition and plunge.